### ISSUE TITLE
Rebuild wheel with TensorRT build options appears to be broken on Jetson   

### ISSUE THREAD

**Issue by isra60** [microsoft/onnxruntime#9378]
**Describe the bug**
I have a Jetson Xavier NX and I want to try the TensorRT EP onnxruntime, so instead of using the standard wheel from the Jetson Model Zoo (https://elinux.org/Jetson_Zoo#ONNX_Runtime) I need to build the entire onnxruntime again with the option of  TensorRT libraries

As reference, I'm using this tutorial https://onnxruntime.ai/docs/build/eps.html#nvidia-jetson-tx1tx2nanoxavier

**System information**
Jetpack 4.6 L4T 
TensorRT 8.0.1

**To Reproduce**
First I uninstall the onnxruntime-gpu package provided by NVidia in the model zoo to avoid conflicts.

> pip3 uninstall onnxruntime-gpu

This is my command line to build onnxruntime wheel using the TensorRT build options

> ./build.sh --config Release --update --build --parallel --build_wheel --build_shared_lib  --use_tensorrt --cuda_home /usr/local/cuda --cudnn_home /usr/lib/aarch64-linux-gnu  --tensorrt_home /usr/lib/aarch64-linux-gnu 

Then the .whl file is generated and I install it 

> pip3 install onnxruntime_gpu_tensorrt-1.10.0-cp36-cp36m-linux_aarch64.whl

Is installed as 
**onnxruntime-gpu-tensorrt 1.10.0**

But when I try to use it...

>  import onnxruntime as rt
>  rt.get_device()

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'onnxruntime' has no attribute 'get_device'
```
As the tutorial says the TensorRT option is experimental but I suppose this should be working right? 
To add more info the wheel provided by NVidia works flawlessly, installed as **onnxruntime-gpu    1.8.0**

**Comment by jywu-msft** [comment]
yes, it should work.
I just tested on a Jetson Nano and it works fine. (I built rel-1.9.1 branch though, but master should work too)

onnxruntime@onnxruntime-nano:~/onnxruntime/build/Linux/MinSizeRel/dist$ pip3 install onnxruntime_gpu_tensorrt-1.9.1-cp36-cp36m-linux_aarch64.whl
Processing ./onnxruntime_gpu_tensorrt-1.9.1-cp36-cp36m-linux_aarch64.whl
Collecting protobuf (from onnxruntime-gpu-tensorrt==1.9.1)
  Using cached https://files.pythonhosted.org/packages/ec/2e/309682e1fe979b17d6fec004e8107a953d1300b93c32c397a9c3ca60322e/protobuf-3.18.1-py2.py3-none-any.whl
Collecting numpy>=1.13.3 (from onnxruntime-gpu-tensorrt==1.9.1)
  Using cached https://files.pythonhosted.org/packages/51/60/3f0fe5b7675a461d96b9d6729beecd3532565743278a9c3fe6dd09697fa7/numpy-1.19.5.zip
Collecting flatbuffers (from onnxruntime-gpu-tensorrt==1.9.1)
  Using cached https://files.pythonhosted.org/packages/3d/d0/26033c70d642fbc1e35d3619cf3210986fb953c173b1226709f75056c149/flatbuffers-2.0-py2.py3-none-any.whl
Building wheels for collected packages: numpy
  Running setup.py bdist_wheel for numpy ... done
  Stored in directory: /home/onnxruntime/.cache/pip/wheels/ee/cd/78/686734467766f26e3d2b42605dce9fdb7d24c1c1d26f2fb8fc
Successfully built numpy
Installing collected packages: protobuf, numpy, flatbuffers, onnxruntime-gpu-tensorrt
Successfully installed flatbuffers-2.0 numpy-1.19.5 onnxruntime-gpu-tensorrt-1.9.1 protobuf-3.18.1

onnxruntime@onnxruntime-nano:~/onnxruntime/build/Linux/MinSizeRel/dist$ python3
Python 3.6.9 (default, Jan 26 2021, 15:33:00)
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
import>>> import onnxruntime as rt
>>> rt.get_device()
'GPU'
